api/gen/swagger.yaml

```diff
@@ -50461,12 +50267,20 @@ paths:
               accounts:
                 type: string
                 format: uint64
+                description: The number of AutoCCTP accounts created.
               transfers:
                 type: string
                 format: uint64
+                description: The number of transfers executed.
               total_transferred:
                 type: string
                 format: uint64
+                description: The total amount transferred.
+            description: >-
+              QueryStatsByDestinationDomainResponse is the response message
+              containing stats for a
+
```

Comment:
looks like there is an extra line break here as well

Reply:
Ditto as above!